lib/igniter/code/module.ex

```diff
@@ -48,6 +48,13 @@ defmodule Igniter.Code.Module do
     )
   end
 
+  # TODO: Decide if this is the correct API and make public
```

Comment:
This seemed like a potentially useful API (given a node, find the containing defmodule), but I didn't want to shoehorn it into this PR. If you'd prefer this not be here, I'm happy to make it a private function where it's used.

Reply:
Seems very useful indeed! Go for it 😁

Reply:
Just to be clear: Should I make it public and document it in this PR, or in a separate PR after this is merged?

Reply:
😅sorry, in this PR is good 😀

Reply:
See [here](https://github.com/ash-project/igniter/pull/111#discussion_r1817875500); plan to do this in another PR since this function isn't necessary for this PR anymore.